README.md

```diff
@@ -87,6 +88,7 @@ manager.start();
 * `connectionUri` - MongoDB URI
 * `databaseName` - name of the database
 * `collectionName` - name of the collection on which change stream listener should be applied
+* `keyName` - name of the key that will be used as partitioning key. Default value is `_id`. Key value is required for every document and should be of type ObjectId.
```

Comment:
You can add info that it doesn't have to be unique.

Reply:
done